.github/workflows/self-approval.yml

```diff
@@ -0,0 +1,41 @@
+# Flow for self-approving the PRs, used by maintainers for cases aligned between them.
+# To trigger it, the maintainer needs to commit and push:
+# $ git commit -m "self-approval" --allow-empty
+
+name: Self Approval
+
+on:
+  - pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  automate:
+    name: Automate
+    runs-on: ubuntu-latest
+    if: ${{ contains(fromJson('["julienfalque", "keradus", "keradus-ci", "kubawerlos", "localheinz", "Wirone"]'), github.actor) }}
```

Comment:
Why it wasn't included in this PR's CI? You're the actor, so it should be triggered?

Reply:
Ah, right. So let's merge 🙂.